src/components/HeroSection/HeroSection.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import styles from "./HeroSection.module.css";
+
+const HeroSection = () => {
+  return (
+    <div
+      className={`g-flex-row g-flex-wrap g-flex-align-center ${styles.heroSection}`}
+    >
+      <section className={`g-flex-column ${styles.heroContent}`}>
+        <h1 className={`${styles.heroContentHeading} fw-700 fs-2-5`}>
+          Why MI STREAM?
+        </h1>
+        <p>
+          Interviews are a crucial part of one's life that decides the career
+          curve of an individual be it for getting admission into prestigious
+          institutions like LBSNAA,IIM ,etc. Or be it while switching a job to
+          another well paying organisation. All of us have to pass through the{" "}
+          <strong>interviews</strong>.
+        </p>
+        <p>
+          At these crucial times,it is always better to be familiar about the
+          interview process and an interactive way to help you perform better in
+          it is by engaging in mock interviews helping you identify your
+          strengths and weaknesses.
+        </p>
+      </section>
+      <section className={`${styles.heroImageContainer}`}>
+        <img src="assets/HeroSectionImage.svg" alt="" />
```

Comment:
can give an appropriate alt message here.

Reply:
Yes it is a dummy category so kept it empty for the time being!!

Reply:
Yes have noticed the change. Made the changes!!